impl/src/main/java/org/jboss/weld/bootstrap/events/AfterTypeDiscoveryImpl.java

```diff
@@ -76,6 +77,7 @@ protected AfterTypeDiscoveryImpl(BeanManagerImpl beanManager, BeanDeploymentArch
     public void addAnnotatedType(AnnotatedType<?> type, String id) {
         checkWithinObserverNotification();
         addSyntheticAnnotatedType(type, id);
+        BootstrapLogger.LOG.addAnnotatedTypeCalled(receiver.getClass().getName(), type.getJavaClass());
```

Comment:
I think `getReceiver().getClass()` should be enough. And we could consider `type.toString()` instead of `type.getJavaClass()`?